### PR TITLE
Automation: Run all pre-commit hooks

### PR DESCRIPTION
**Pull request by SuyashSalampuria**
#6283 
Edited flake8 config for running pre-commit run --all-files alongside black

### CODE REVIEW

**Comment by SuyashSalampuria** [comment]
@armenzg 
Please review

File: setup.cfg [hunk 1]
```diff
@@ -4,7 +4,7 @@ exclude = */.*/,.*/,__pycache__,node_modules
 # E203 whitespace before ':'
 # E231: missing whitespace after ','
 # E501: line too long
-extend_ignore = E129,E203,E231,E501
+ignore = E101, E111, E114, E115, E116, E117, E121, E122, E123, E124, E125, E126, E127, E128, E129, E131, E133, E2, E3, E5, E501, E701, E702, E703, E704, W1, W2, W3, W503, W504
```
Comment (armenzg) [inline setup.cfg:7]: What's the difference between `ignore` and `extend_ignore`?

Comment (armenzg) [reply setup.cfg:7]: I'm curious. Can we leave it as it was? (so it's shorter list)

Comment (SuyashSalampuria) [reply setup.cfg]: Yes, it must be left as it was
ignore overrides the list of files to be left
extend-ignore extends the list

**Comment by SuyashSalampuria** [comment]
Did the required changes @armenzg

**Pull request merged**